### PR TITLE
oryoki 0.0.3 (new cask)

### PR DESCRIPTION
**Pull request by dunn**
A minimalist web browser.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/oryoki.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'oryoki' do
+  version '0.0.3'
+  sha256 '51f88440e062e23abd7334bd20a9d4d0997cd6dc73f4962f02117c820e10ea54'
+
+  # github.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/oryoki.rb]: Too imprecise: `github.com/thmsbfft/oryoki`.

File: Casks/oryoki.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  url "https://github.com/thmsbfft/oryoki/releases/download/#{version}/Oryoki-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/oryoki.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/oryoki.rb (hunk 1 above)
Comment (dunn) [inline Casks/oryoki.rb]: Oh, got it; I just took the text from the audit.

File: Casks/oryoki.rb (hunk 2 above)
Comment (dunn) [inline Casks/oryoki.rb]: Done!

File: Casks/oryoki.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/oryoki.rb]: I know, it’s understandable.

**Comment by vitorgalvao** [comment]
Looks great! Thank you.

**Pull request merged**